Tryouts/Plugins/ApplicationPlugins/process explorer/super-rpc/README.md

```diff
@@ -0,0 +1,172 @@
+# SuperRPC
```

Comment:
Please open a PR on the repository of this project to merge these changes.

Reply:
Deleted the library, now depending as a npm package due the issue was solved and patched. (callId was handled as type number instead of string)